silkworm/silkrpc/core/gas_price_oracle.cpp

```diff
@@ -64,7 +64,7 @@ Task<intx::uint256> GasPriceOracle::suggested_price(uint64_t block_number) {
     co_return price;
 }
 
-Task<void> GasPriceOracle::load_block_prices(uint64_t block_number, uint64_t limit, std::vector<intx::uint256>& tx_prices) {
+Task<void> GasPriceOracle::load_block_prices(BlockNum block_number, uint64_t limit, std::vector<intx::uint256>& tx_prices) {
```

Comment:
The same change should be applied also in `gas_price_oracle.hpp`

Reply:
ok